sklearn/model_selection/tests/test_split.py

```diff
@@ -783,6 +783,17 @@ def train_test_split_pandas():
         assert_true(isinstance(X_test, InputFeatureType))
 
 
+def train_test_split_sparse():
+    # check that train_test_split converts scipy sparse matrices to csr
```

Comment:
Maybe say that this is too be consistent with the documentation: we did do this by design, did we?